scripts/us_census/pep/us_pep_sexrace/postprocess.py

```diff
@@ -0,0 +1,370 @@
+    mcf_template = """Node: dcid:{}
+typeOf: dcs:StatisticalVariable
+populationType: dcs:Person
+{}{}
+statType: dcs:measuredValue
+measuredProperty: dcs:count
+"""
+
+    final_mcf_template = ""
+    for sv in statvar_names:
+        if "Total" in sv:
+            continue
+        if "Year" in sv:
+            continue
+        if "geo_ID" in sv:
+            continue
+        gender = ''
+        race = ''
+        sv_prop = sv.split("_")
+        for prop in sv_prop:
+            if prop in ["Count", "Person"]:
+                continue
+            if "Male" in prop or "Female" in prop:
+                gender = "gender: dcs:" + prop
+            else:
+                race = "race: dcs:" + prop + "\n"
+        final_mcf_template += mcf_template.format(sv, race, gender) + "\n"
+    # Writing Genereated MCF to local path.
+    if flag1 == 1:
+        with open(_CODEDIR + "/output_files/final/" + "sex_race.mcf",
+                  'w+',
+                  encoding='utf-8') as f_out:
+            f_out.write(final_mcf_template.rstrip('\n'))
+    elif flag1 == 2:
+        with open(_CODEDIR + "/output_files/final/" +
+                  "sex_race_aggregate_before.mcf",
+                  'w+',
+                  encoding='utf-8') as f_out:
+            f_out.write(final_mcf_template.rstrip('\n'))
+    elif flag1 == 3:
+        with open(_CODEDIR + "/output_files/final/" +
+                  "sex_race_aggregate_after.mcf",
+                  'w+',
+                  encoding='utf-8') as f_out:
+            f_out.write(final_mcf_template.rstrip('\n'))
+    else:
+        with open(_CODEDIR + "/output_files/final/" + "sex_race_aggregate_national_after.mcf",\
+             'w+', encoding='utf-8') as f_out:
+            f_out.write(final_mcf_template.rstrip('\n'))
+
+
+def generate_tmcf(df_cols: list, flag2: int) -> None:
+    """
+            This method generates 3 TMCF file w.r.t
+    dataframe headers and defined TMCF template
+    1. Sex_Rcae.tmcf : for the files which are processed as is.
+    2. sex_race_aggregate.tmcf : for the files which
+        are having Count Person Male/Female aggregated.
+    3. sex_race_aggregate_state_2010_2020.tmcf : for the files
+        which are aggregated from different geo granularity.
+
+    Args:
+        df_cols (list) : List of DataFrame Columns
+        flag2 (int) : flag value helps in generating
+        output files. Possible values are 1,2 and 3.
+        1 - generate tmcf for files which are processed as is
+        2 - generate tmcf for files which are having
+        Count_Person_Male and Count_Person_Female aggregated
+        3 - generate tmcf for files which are aggregated
+        from different geo granulartiy.
+
+    Returns:
+        None
+    """
+
+    tmcf_template = """Node: E:postprocess->E{}
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:{}
+measurementMethod: dcs:{}
+observationAbout: C:postprocess->geo_ID
+observationDate: C:postprocess->Year
+observationPeriod: \"P1Y\"
```

Comment:
Since this is within """...""", we don't need the back-slash (\) to escape the double quote?

Reply:
Removed the back-slash.